net-analyzer/ipsumdump/ipsumdump-1.85.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Simple TCP/IP Dump summarizer/analyzer"
+HOMEPAGE="http://read.seas.harvard.edu/~kohler/ipsumdump/"
+SRC_URI="http://read.seas.harvard.edu/~kohler/ipsumdump/ipsumdump-1.85.tar.gz"
+
+LICENSE="the-Click-license"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+ipv6 +nanotimestamp"
+
+DEPEND="dev-lang/perl
```

Comment:
I only checked this quickly, but I suspect perl/texinfo are only used at build-time for generating the man pages.

configure does reference expat, but I wasn't able to find any actual usage of it anywhere.